typingtraining

```diff
@@ -11,18 +11,22 @@ from lib.tracker import Tracker
 def blocking_exit(stdscr):
     stdscr.addstr('To proceed press "BACKSPACE" key...', curses.color_pair(3))
     stdscr.refresh()
-    while 'KEY_BACKSPACE' != stdscr.getkey():
-        pass
+    while True:
+        print(stdscr.getkey())
 
 
-def main(content):
+def main(content, non_stop_mode):
     def main_inner(stdscr):
         if curses.has_colors():
             # height, width = stdscr.getmaxyx()
             lib = Library()
             # lib.createdefault()
             tracker = Tracker(content(lib))
-            tracker.start_tracking(stdscr)
+            repeats_counter = int(non_stop_mode)
```

Comment:
Consider removing this 'cast', this should be int anyway. Maybe merge this variable with `non_stop_mode`?

Reply:
While removing cast is possible, merging is not becouse non_stop_mode is variable from Main function which returns main_inner function as return value. Afterwards this main_inner function is send as argument to curses.wrapper. By that time non_stop_mode is const value written inside main_inner, if we merge those variables main_inner would by trying to access non_stop_mode from inside main_inner and fail in doing so.